libs/README.md

```diff
@@ -0,0 +1,68 @@
+# JavaCard Libraries
+
+Local `*.jar` dependencies repository.
+
+You can add here local dependencies if there are not available on the 
+Maven central repository or you are not willing to use those.
+
+If there is a `test.jar` file you can add it as a dependency
+by adding the following line to the `dependencies {}` block.
+
+```gradle
+compile name: 'test'
+```
+
+This works only for JAR files placed right in the `/libs` directory (flat hierarchy).
+The artifact group is ignored, artifact is searched just by the name.
+ 
+For subdirectories you have to use the `files()` or `fileTree` as demonstrated below.
+
+## Custom JCardSim
```

Comment:
I don't think we use jcardsim yet, can you clean up this readme to contain only the relevant information?

Reply:
The `README.md` has been updated.